test/parallel/test-https-connect-address-family.js

```diff
@@ -7,20 +7,22 @@ if (!common.hasIPv6)
   common.skip('no IPv6 support');
 
 const assert = require('assert');
+const fixtures = require('../common/fixtures');
```

Comment:
Not needed, available as `common.fixtures`

Reply:
Hrm. That doesn't seem to work. I also don't see `common.fixtures` in any other test. There's `common.fixturesDir`, though that just points to the directory. Is that plus `fs.readFileSync` plus path-munging preferred over `fixtures.readKey`?

Reply:
This seems fine as is. It matches what people were instructed to use in the latest Code & Learn.

Reply:
Sorry, you're both right.  I thought (don't know why) that common.js re-exported everything from fixtures.js.